src/main/java/it/gov/pagopa/common/web/exception/ValidationExceptionHandler.java

```diff
@@ -35,7 +36,7 @@ public ErrorDTO handleValidationExceptions(
         ErrorManager.getRequestDetails(request), message);
     log.debug("Something went wrong while validating http request", ex);
 
-    return new ErrorDTO(HttpStatus.BAD_REQUEST.value(), message);
+    return new ErrorDTO(IbanConstants.ExceptionCode.INVALID_REQUEST, message);
```

Comment:
this is a "common" class! remove the dependency from the "iban" class: eg we could define a @Bean templateValidationErrorDTO used just to provide default configuration in order to set here the "code"